webapp/webapp/webapp/static/admin/js/domains.js

```diff
@@ -421,9 +486,28 @@ $(document).ready(function() {
                 icon: 'fa fa-'+data.icon,
                 opacity: 1,
                 type: data.type
-        });
+        });       
```

Comment:
Please, could you remove tailing whitespaces?
```suggestion
        });
```